examples/chef/nrfconnect/CMakeLists.txt

```diff
@@ -45,6 +41,7 @@ find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
 
 project(chip-nrfconnect-chef-example)
 
+include(${CHIP_ROOT}/config/nrfconnect/app/check-sysbuild-use.cmake)
```

Comment:
Don't you need to add `sysbuild` directory to chef app as well?

Reply:
Yes, definitely, I must have missed this one.